tests/h/views/activity_test.py

```diff
@@ -927,6 +927,11 @@ def group(factories):
     factories.Group()
 
     group = factories.Group()
+
+    # Add the group's creator as a member, as would normally be the case for
+    # (non-publisher) groups.
+    group.members.append(group.creator)
+
```

Comment:
Probably not right now. We can look at renaming this when we add a new group type with its own factory. Thanks!